### PR TITLE
81763 home aem query

### PR DESCRIPTION
**Pull request by DaceyTom2**
### Description
Added query and mapper for AEM home

List of proposed changes:
- Query added
- Mapper added

### What to test for/How to test
From home the content object returned from mapHome can be logged

### Additional Notes
I haven't added test for the mapper yet as it is expected to change shortly
Mapper can also use some cleanup/refactoring and am looking for input from other developers

### CODE REVIEW

File: graphql/client.js [hunk 1]
```diff
@@ -13,11 +13,9 @@ const client = new ApolloClient({
...
 
 export default async function (AEMQuery) {
-  return client
-    .query({
-      query: AEMQuery,
-    })
-    .catch((error) => {
-      console.log(error)
-    })
+  return client.query({
```
Comment (nibivi77) [inline graphql/client.js:16]: Should we have a `catch()` here? Good opportunity for our logger as well.

File: pages/home.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default function Home(props) {
 }
 
 export async function getStaticProps({ locale }) {
-  const content = await fetchContent()
+  const content = await getHomeContent()
```
Comment (nibivi77) [inline pages/home.js:50]: Should we pass the content into the cards? or do this in a separate PR?

File: graphql/client.js (hunk 1 above)
Comment (DaceyTom2) [reply graphql/client.js:16]: I'd rather not catch the error here since we won't know what page is throwing it. I'll add at the page level
